suite/tests/api/ir_aarch64.c

```diff
@@ -5336,8 +5336,23 @@ test_asimddiff(void *dc)
     test_instr_encoding(dc, OP_umull2, instr);
 }
 
+/* Macro expansion generates sequence of tests for creating cache instructions
+ * which are aliases of SYS instruction variants:
+ * DC ZVA, Xt        DC CVAC, Xt        DC CVAU, Xt        DC CIVAC Xt
+ * DC IVAC Xt        DC ISW Xt          DC CSW Xt          DC CISW Xt
+ * IC IVAU, Xt       IC IALLU           IC IALLUIS
+ */
+
+#define SYS_CACHE(cache, op)                                             \
```

Comment:
A name that reflects the loop may help: TEST_SYS_CACHE_ALL_REGS or sthg

Reply:
Done.